client/src/components/App/Content/InvestigationForm/TabManagement/PersonalInfoTab/PersonalInfoTab.tsx

```diff
@@ -566,13 +383,13 @@ const PersonalInfoTab: React.FC<Props> = ({ id }: Props): JSX.Element => {
                                                             value={props.value}
                                                             onChange={(event, newValue) => {
                                                                 newValue && getEducationSubOccupations(newValue.value.displayName);
-                                                                setSubOccupationName('');
+                                                                methods.setValue(PersonalInfoDataContextFields.EDUCATION_OCCUPATION_CITY, '');
                                                                 props.onChange(newValue ? newValue.value.displayName : '')
                                                             }}
                                                             renderInput={(params) =>
                                                                 <TextField
                                                                     {...params}
-                                                                    error={methods.errors[PersonalInfoDataContextFields.EDUCATION_OCCUPATION_CITY]}
+                                                                    error={(methods.errors[PersonalInfoDataContextFields.EDUCATION_OCCUPATION_CITY] as unknown) as boolean}
```

Comment:
isn't is simpler to do something like Boolean(X) or methods.errors.hasOwnProperty(X)?